packages/ds-healthcare-gov/src/components/Header/__snapshots__/defaultMenuLinks.test.js.snap

```diff
@@ -12,6 +12,7 @@ Object {
       "label": "My profile",
     },
     Object {
+      "ariaLabel": "CuidadoDeSalud.gov en Español",
```

Comment:
I'm a little curious what this snapshot test is actually testing

Reply:
🤔 Looks like a lot of snapshot tests around hgov Header and Footer look like this.